code/game/gamemodes/wizard/artefact.dm

```diff
@@ -181,6 +181,79 @@
 			qdel(src)
 		return
 	..()
+//Why did i add this....-Fethas
+/obj/item/weapon/veilrender/honkrender //HONK!!
+	name = "honk render"
+	desc = "A wicked curved 'blade' of honk origin, recovered from the ruins of a vast circus."
+	icon_state = "clownrender"
+
+/obj/item/weapon/veilrender/honkrender/attack_self(mob/user as mob)
+	if(charged)
+		new /obj/effect/rend/honk(get_turf(usr))
+		charged = 0
+		visible_message("\red <B>[src] hums with power as [usr] deals a blow to depression itself!</B>")
```

Comment:
Color macros aren't good. \red, \blue, anything like that should be avoided. Especially since this has `<b>` already, just change the `\red and <b>` tags to `<span class='danger'>`. Be sure to end it at the end with `</span>`